zebra-rpc/src/methods/get_block_template_rpcs/config.rs

```diff
@@ -9,7 +9,7 @@ use zebra_chain::transparent;
 #[serde(deny_unknown_fields, default)]
 pub struct Config {
     /// The address used for miner payouts.
-    /// Zebra currently only supports single-signature P2SH transparent addresses.
+    /// Zebra currently only supports single-signature P2SH and P2PKH transparent addresses.
```

Comment:
This is my mistake, I thought I'd got all of them

```suggestion
    /// Zebra currently only supports P2SH and P2PKH transparent addresses.
```